nav/virtual_layers/src/virtual_layer.cpp

```diff
@@ -14,27 +14,45 @@ VirtualLayer::VirtualLayer() {
   listener.waitForTransform("/odom", "/left_camera_link_optical", ros::Time(0), ros::Duration(60.0));
 }
 
+geometry_msgs::Point VirtualLayer::transform_from_camera_to_odom(double x, double y, double z) {
+  geometry_msgs::PoseStamped new_pose = geometry_msgs::PoseStamped();
+  new_pose.header.frame_id = "left_camera_link_optical";
```

Comment:
@spencerteetaert do you forward the header in the CV stuff? 

If yes, @1999michael use the frame_id from that, otherwise update it to be whatever the real zed expects (its 100% a diff frame)